carapace-server/src/test/java/org/carapaceproxy/server/RequestsLoggerTest.java

```diff
@@ -485,19 +479,19 @@ public void testWithServer() throws Exception {
         EndpointKey key = new EndpointKey("localhost", wireMockRule.port());
 
         ConnectionsManagerStats stats;
-        try (HttpProxyServer server = HttpProxyServer.buildForTests("localhost", 0, mapper, tmpDir.newFolder());) {
-            server.getCurrentConfiguration().setAccessLogPath(tmpDir.getRoot().getAbsolutePath()+"/access.log");
+        try ( HttpProxyServer server = HttpProxyServer.buildForTests("localhost", 0, mapper, tmpDir.newFolder());) {
```

Comment:
This is a bad formatting issue... Please fix editor formatting properties or disable/avoid auto-formatting